tests/Unit/UserIdentityModelTest.php

```diff
@@ -72,4 +75,52 @@ public function testCreateCodeIdentityThrowsExceptionIfUniqueCodeIsNotGot(): voi
             $generator
         );
     }
+
+    public function testForceMultiplePasswordReset(): void
+    {
+        /**
+         * @phpstan-var Fabricator
+         */
+        $fabricator = new Fabricator(UserIdentityModel::class);
+        $fabricator->create(10);
+
+        /**
+         * @phpstan-var UserIdentityModel
+         */
+        $identities = model(UserIdentityModel::class);
+        $result     = $identities->select('user_id')->findAll();
+        $userIds    = [];
+
+        foreach ($result as $row) {
+            $userIds[] = $row->user_id;
+        }
+
+        $response = $identities->forceMultiplePasswordReset($userIds);
+        $this->assertTrue($response);
+    }
+
+    public function testforceGlobalPasswordReset(): void
```

Comment:
This is not a unit test.
Please move to other test class.

Reply:
Done, please check.